### ISSUE TITLE
What happened with the commit history?

### ISSUE THREAD

**Issue by wundo** [enten/angular-universal#62]
What happened with the commit history?

**Comment by enten** [comment]
From the begining (2017), this repository used package [udk](https://github.com/enten/udk) to allow multicompilation in once webpack run. In that way, we have a single process with a single run of webpack to build browser bundle of the angular app, server bundle of the angular app and the node.js to serve the angular app and do the SSR. In addition, that allow HMR on browser and servide side.

To make everything works udk throught angular releases: that required some developpement on udk package.

With webpack5: the development needed on udk package to make everything works is important. The main purpose of udk is multicompilation enhancement. But we found that multicompilation with webpack is rarely used. So development of udk is currently suspended.

Despite udk, I think that this repository can continue to live without advantage of udk.

I made major changes and I thought that the commit history has no sense (because udk is now suspended): so I've decided to start the history for scratch.

My bad if you have a fork which still used builder from udk (as precious version of the repo): but if you want to upgrade to angular 12, you must consider migrate your repo to copy the structure of the current master branch.
